styleguide.md

```diff
@@ -0,0 +1,138 @@
+# Documentation Style Guide
+
+**Admin** - proper case
+
+**Admin API** - proper case
+
+**Admin GUI** - use *Kong Manage*r instead
+
+**ampersands** - always use “and” in content, only use “&” in titles with two items that have always had it or in product names
+
+**API** - all upper case; if referring to an entity, use Services and Routes instead
+
+**Basic Auth, Basic Authentication** - proper case, abbreviate if the context permits. Only use lowercase with a hyphen in code/config samples
+
+* Bad: Options include LDAP Auth and basic auth.
+* Good: Options include LDAP Auth and Basic Auth.
+* Bad: Ensure that admin_gui_auth is set to basic auth.
+* Good: Ensure that admin_gui_auth is set to basic-auth.
+* Good: Ensure that admin_gui_auth = basic-auth.
+
+
+**click** - not “click on”; specify what is being clicked
+
+* *Bad*: Click on “Admins”
+* *Bad*: Look for one that says “Admins” and click
+* **_Good_: Click the “Admins” button**
+
+**cURL** - not CURL or curl
+
+**e.g**. - indicates an in-exhaustive list of examples, thus should not conclude with “etc.”; if offering a clarification, rather than an example, use “i.e.”. Following the Chicago Manual, follow each with a comma.
```

Comment:
I know Google's autocorrect catches this, but it's wrong: https://www.thefreedictionary.com/inexhaustive

```suggestion
**e.g**. - indicates an inexhaustive list of examples, thus should not conclude with “etc.”; if offering a clarification, rather than an example, use “i.e.”. Following the Chicago Manual, follow each with a comma.
```